src/core/utils/common.spec.tsx

```diff
@@ -10,7 +10,7 @@ import {
   getTabbableElements,
   getUserColor,
   getWindow,
-} from './common';
+} from '.';
```

Comment:
I would say to split tests to separate files too.

Reply:
Done